### ISSUE TITLE
How I can use map-directions with markers in clusters ?

### ISSUE THREAD

**Issue by dev0088** [codempireio/react-native-maps-clusterize#37]
I want to use react-native-maps-directions with this react-native-cluster-map package.

I was using `react-native-maps` and `react-native-maps-directions`.
After I changed `react-native-maps` to `
react-native-cluster-map`, I had this issue.
```
TypeError: Cannot read property 'coordinate' of undefind
```

I think this `react-native-cluster-map` allow only `Marker` child node and don't allow other child node, yet.
Please solve this map direction problem in this ClusterMap node.

**Comment by dimagreat** [comment]
Resolved in #38
Pass isOutsideCluster={true} prop for you non Markers elements

**Comment by dev0088** [comment]
I need to use Markers elements and MapDirection elements in child nodes.
So I added the MapDirection elements in `priorityMarker` variable. It's working. 
```
<ClusterMap
   initialRegion={INITIALIZE_REGION}
   region={region}
   style={styles.map}
   provider={PROVIDER_GOOGLE}
   mapType={"standard"}
   showsUserLocation={true}
   showsCompass={true}
   rotateEnabled={true}
   loadingEnabled={true}
   showsBuildings={true}
   pitchEnabled={true}
   onMapReady={() => this.onMapReady(region)}
   onUserLocationChange={this.onUserLocationChange}
   zoomTapEnabled={false}
   renderClusterMarker={this.renderCustomClusterMarker}
   ref={c => this.mapView = c}
   priorityMarker={this.renderMapViewDirection()}>

   {this.renderMarkers()}

</ClusterMap>
```
